src/index.js

```diff
@@ -1,22 +1,32 @@
 import angular from 'angular';
-import {ngAnimate} from 'angular-animate';
-import {ngSanitize} from 'angular-sanitize';
+import { ngAnimate } from 'angular-animate';
+import { ngSanitize } from 'angular-sanitize';
 import 'angular-ui-bootstrap';
 
 import 'angular-ui-router';
+import 'angular-translate';
 import routesConfig from './routes';
 
-import {hello} from './app/hello';
-import {test01} from './test01/app';
-import {test02} from './test02/app';
+import { hello } from './app/hello';
+
+import { buttonService } from './test01/services/buttonService';
+import { windowResizeDirective } from './test01/directives/windowResize';
+import { buttonResizeDirective } from './test01/directives/buttonResize';
+import { test01 } from './test01/app';
+import { test02 } from './test02/app';
 
 import './index.scss';
 
 export const app = 'app';
 
 angular
-  .module(app, ['ngAnimate', 'ngSanitize','ui.router', 'ui.bootstrap'])
+  .module(app, ['ngAnimate', 'ngSanitize', 'ui.router', 'ui.bootstrap', 'pascalprecht.translate'])
   .config(routesConfig)
   .component('app', hello)
   .component('test01', test01)
   .component('test02', test02);
+
+buttonService();
+windowResizeDirective();
+buttonResizeDirective();
```

Comment:
This may not be the best way to declare these angular services and directives, but given that I have never seen an angular application set up this way (importing components using constants), this was the cleanest method of declaring them that I could think of.